cmd/client-admin.go

```diff
@@ -79,11 +79,11 @@ func newAdminFactory() func(config *Config) (*madmin.AdminClient, *probe.Error)
 			var transport http.RoundTripper = &http.Transport{
 				Proxy: http.ProxyFromEnvironment,
 				DialContext: (&net.Dialer{
-					Timeout:   30 * time.Second,
-					KeepAlive: 30 * time.Second,
+					Timeout:   60 * time.Second,
```

Comment:
we want to keep the dial timeout for 30secs to quickly know if the host is reachable, 60secs is too high IMO @wlan0